go/client/cmd_team_add_member.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+type CmdTeamAddMember struct {
+	libkb.Contextified
+	team     string
+	username string
+	role     keybase1.TeamRole
+}
+
+func newCmdTeamAddMember(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "add-member",
+		ArgumentHelp: "<team name> --user=<username> --role=<owner|admin|writer|reader>",
```

Comment:
I was just following the MVP sandbox doc.  I think this is fine for now.  

There is a service endpoint for the full ChangeMembership where multiple can be added/moved/removed in one call.